src/app/sellers/store/effects/sellers.effects.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { of } from 'rxjs';
+import { catchError, map, mergeMap, switchMap } from 'rxjs/operators';
+
+import { SellersActionTypes, Load, LoadSellersSuccess, LoadFail, LoadPageSuccess } from '../actions/sellers.actions';
+
+import { SellerService } from '../../../core/services/seller.service';
+import { SellerObjectModel, SellerPayloadModel } from '../../../core/models/seller.model';
+
+
+@Injectable()
+export class SellersEffects {
+  @Effect()
+  loadSellers$ = this.actions$.pipe(
+    ofType<Load>(SellersActionTypes.Load),
+    map(action => action.payload),
+    switchMap((params: any) =>
+      this.sellerService
+        .getSellers(params)
+        .pipe(
+          map( (res: SellerPayloadModel) => {
+            // this.updateElements(res);
```

Comment:
Este método lo que hace es convertir los campos de cliente (tipo teléfono, sexo, etc). Por eso no te mostraba información, porque lo que retorna el BE para los vendedores es diferente y no tiene esas mismas propiedades